extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1399,6 +1399,13 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   onVideoVisibilityUpdate_(event) {
+    // AmpDoc visibility updates are hadnled by the PAUSED state. This method
```

Comment:
nit: s/hadnled/handled

Reply:
2quick4me

Reply:
You have been  ![image](https://user-images.githubusercontent.com/1492044/68979727-52143400-07cc-11ea-9672-7786bf1ba160.png)